src/data/pyproject.toml

```
@@ -22,6 +22,8 @@ dependencies = [
     "scipy==1.10.1",
     "jsonlines==3.1.0",
     "datasketch<2.0.0",
-    "beautifulsoup4==4.12.2"
+    "beautifulsoup4==4.12.2",
```

```suggestion
    "beautifulsoup4==4.12.2, < 5",
```
https://peps.python.org/pep-0631/